ex3/ex3.c

```diff
@@ -10,6 +10,17 @@
 int main(void)
 {
     // Your code here
-
+    pid_t rc = fork();
+    if (rc == 0){
+        printf("hello\n");
+        exit(1);
```

Comment:
Using `exit(1)` here indicates that the program terminated unsuccessfully. In this case the program is working as expected, so you probably don't want to do that.  

It might seem counterintuitive, but in C (and programming in general), a successful program terminates with the code `0`. This is why we have `return 0` at the bottom of our `main` function. The way I think of it is that when we write `exit(0)` we are saying that the program ran with 0 errors.